wip-0009.md

```diff
@@ -61,21 +61,46 @@ Upon reception of multiple candidates:
 2. Block candidates from active identities are prioritized over those from non-active identities.
 3. If there are still multiple candidates, the one with the lowest VRF value is chosen.
 
-With the previous changes, the newcomer mining probability is bounded as initially designed in [RandPoe] while the fair distribution among those identities that have proven some contribution to the network's service and security still applies.
+With these proposed changes to the block candidate selection algorithm, the newcomer mining probability is now bounded as initially designed in [RandPoe] while the fair distribution among those identities that have proven some contribution to the network's service and security still applies. As an illustrative example using the same network conditions as in the previous section, with the new validation rules in place the newcomer mining probability would go down to almost 5%.
+
+
+## Backwards compatibility
+
+### Consensus
+
+This proposal introduces a significant change in the block candidate selection algorithm. Therefore there is enough evidence to assert that it is not backward compatible from the consensus perspective and that no cooperative chain construction would be possible among two nodes from which one adopted these changes while the other did not.
+
+As it is considered best practice in such situations, an *Adoption Plan* is hereby proposed.
+
+
+### Examples and Documentation
+
+The algorithm affected by this proposal is internal to the block chain protocol, and it does not trigger any update on existing examples or documentation.
+
+
+### Libraries and Clients
+
+The algorithm affected by this proposal is internal to the block chain protocol, and it does not require any update on existing libraries or clients.
 
 
 ## Reference Implementation
 
 The business logic described in the specification has been implemented in [witnet-rust] by [pull request #1829][witnet/witnet-rust#1829].
 
 
+## Adoption Plan
+
+The changes proposed herein have been already implemented in [witnet-rust] and the activation of the new consensus rule is scheduled as a community-coordinated hard fork to take place on March 16 2021 at 9 am UTC, as proposed by [pull request #1829][witnet/witnet-rust#1829] and [widely announced to the community][announcement] in a timely manner.
+
+
 ## Acknowledgments
 
 This proposal has been cooperatively discussed and devised by many individuals from the Witnet development community [witnet/witnet-rust#1827].
 
 Special thanks to [Luis Rubio][lrubiorod] for the reference implementation in Rust.
 
 
+[announcement]: https://medium.com/@aesedepece/359ff11dfdbe
```

Comment:
I'm not sure if this link will work once the article is published, as it will be moved from my namespace to the foundation's.

I think we can go with this:
```suggestion
[announcement]: https://medium.com/witnet/359ff11dfdbe
```